src/bash_autocomplete.sh

```diff
@@ -15,8 +15,8 @@ function _kw_autocomplete()
 
   kw_options['kw']='init build deploy bd diff ssh vars codestyle self-update
                     maintainers kernel-config-manager config remote explore
-                    pomodoro report device backup debug mail env patch-hub
-                    clear-cache drm vm version man help'
+                    pomodoro profiler report device backup debug mail env
+                    patch-hub clear-cache drm vm version man help'
```

Comment:
I feel that profiler as an option inside kw does not make a lot of sense... I mean, I can see a user thinking that a profiler is something they can use to profile a kernel performance. Tbh, I'm more inclined to make this profiler an external script... maybe something in a folder named `tools`;

Reply:
Agreed. Done it, but in the already present `scripts` directory rather than creating a new directory. Chose this approach as `scripts/kwreview.sh` is also tool that isn't for kw end-user.